locales/en.json

```diff
@@ -0,0 +1,126 @@
+{
+	"language_name": "English",
```

Comment:
I mean we coded the rest of the site but I just thought it was more fitting so we could credit translators for their work (given that they're not part of Dynastic)